pkgs/applications/misc/xmrig/xmrig-cuda.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, autoAddOpenGLRunpathHook
+, fetchFromGitHub
+, cmake
+, pkgs
+, tree
+, cudaArch ? 61
```

Comment:
It would seem that upstream accepts a semicolon-separated list of architectures: https://github.com/xmrig/xmrig-cuda/blob/f04d875daffc1725e60511fff609ab19555c7a84/cmake/CUDA.cmake#L52

Also, can move this down to the `let ... in` and use the default list of architectures from `cudaPackages.cudaFlags`.
Here's an example of generating a semicolon-separated list of architectures, represented as double digit numbers without any periods: https://github.com/NixOS/nixpkgs/blob/6ed024336b9f28e05dda6b2102effad0d717a30c/pkgs/development/libraries/science/math/faiss/default.nix#L101

Again, I know it's not ideal, but it's what we have right now